tests/cli/rfhub_importer.py

```diff
@@ -130,6 +132,14 @@ def test_create_collection_should_return_collection(self):
         result = self.rfhub_importer.create_collection(FIXTURE_PATH / 'SingleClassLib' / 'SingleClassLib.py')
         self.assertDictEqual(EXPECTED_COLLECTION, result)
 
+    def test_create_collections_should_retunr_empty_list_on_data_error(self):

```

Comment:
spelling errors in names of the two new test methods

Reply:
Corrected